### ISSUE TITLE
why you don't wrap all conv in CAM with spectral norm ?

### ISSUE THREAD

**Issue by phucduocdoan** [ufownl/cycle_gan#2]
can you explain for me, because spectral norm should be wrap in all conv layers. i am trying to re-implement using pytorch, so I want to understand if it's really necessary and how it affects training stability.

**Comment by ufownl** [comment]
The kernel size of conv layers in CAM is 1x1, they can be considered as dense layers. But I'm unsure if this is the right way to do it, just that the result doesn't look bad. I didn't have much time and devices to try another way at that time, maybe you can try to compare the two ways.

**Comment by phucduocdoan** [comment]
I tested two versions in pytorch: one with SN in the CAM and one without it. From my tests, adding SN to the CAM made the model work worse.
I got good results using CAM without SN, but there are still some problems when generating faces that are not looking straight.
May I ask if the hyperparameters you used for training are the same as in your public code, or did you use different settings during training?

**Comment by ufownl** [comment]
Just some usual tricks, such as decreasing the learning rate with training. I didn't do a rigorous hyperparameter search due to the lack of time and device, so that may not be optimal.
